src/store/index.js

```diff
@@ -1,8 +1,30 @@
 import { configureStore } from "@reduxjs/toolkit";
 import authenticationReducer from "../features/authenticationSlice.js";
+import postReducer from "../features/postSlice.js";
+import {
+  combineReducers,
+ 
+  getDefaultMiddleware,
+} from '@reduxjs/toolkit';
+import {
+  persistStore,
+  persistReducer,
+  FLUSH,
+  REHYDRATE,
+  PAUSE,
+  PERSIST,
+  PURGE,
+  REGISTER,
+} from 'redux-persist';
```

Comment:
This looks new!! Would explore it out. Do attach a reference in this thread related to using this package